dask/dataframe/groupby.py

```diff
@@ -1984,6 +1990,65 @@ def rolling(self, window, min_periods=None, center=False, win_type=None, axis=0)
             axis=axis,
         )
 
+    def fillna(self, value=None, method=None, limit=None, axis=None):
+        """Fill NA/NaN values using the specified method.
```

Comment:
Does the `@derived_from` decorator not work here?

Reply:
It does, but since we don't support all the `value` types and all keyword arguments, I decided to create a custom docstring -- which copies the pandas' version, just with some edits.